dist/main.css

```diff
@@ -0,0 +1,367 @@
+* {
+    margin: 0;
+    padding: 0;
+    box-sizing: border-box;
+}
+
+html,
+body {
+    font-family: 'Montserrat';
+    background-color: #FFF5F0;
+    overflow-x: hidden;
+}
+@media  all and (min-width: 989px) {
+    
+/* Header items */
+.header__navbar {
+    display: flex;
+    flex-direction: row;
+    gap: 5px;
+    margin: 10px;
+}
+.menu-btn {
+    display: none;
+}
+/* header menu */
+.header__menu {
+    height: 91px;
+    flex-grow: 6;
+    display: flex;
+    gap: 60px;
+    justify-content: center;
+    align-items: center;
+
+}
+
+.header__items {
+    display: flex;
+    gap: 20px;
+    list-style: none;
+}
+
+.header__link {
+    text-decoration: none;
+    font-size: 14px;
+    color: black;
+}
+
+.header__logo {
+    font-weight: bolder;
+}
+
+.header__menu-btns {
+    height: 91px;
+    flex-grow: 4;
+    display: flex;
+    flex-flow: row wrap;
+    gap: 20px;
+    justify-content: start;
+    align-content: center;
+}
+
+/* Header section */
+.header__body {
+    display: grid;
+    height: 60vh;
+    padding-top: 2rem;
+    grid-template-columns: repeat(2, 1fr);
+}
+
+.header__content {
+    display: grid;
+    place-content: center;
+}
+
+.sub-title {
+    font-size: 12px;
+    line-height: 24px;
+    letter-spacing: 0.1px;
+    font-weight: 700;
+    color: #1BA0AB;
+}
+
+.header__text {
+    padding: 25px 0;
+    font-weight: 700;
+    font-size: 38px;
+    letter-spacing: 2px;
+    line-height: 45px;
+    color: #252B42;
+}
+
+.header__sub-text {
+    color: #737373;
+    font-weight: 500;
+    size: 20px;
+    line-height: 30px;
+    letter-spacing: 0.2px;
+}
+
+.header__btns {
+    display: flex;
+    gap: 10px;
+    padding-top: 14px;
+}
+
+.btn--bordered {
+    border: 1px solid #1BA0AB;
+}
+
+/* header buttons and modifiers */
+.btn {
+    color: #1BA0AB;
+    font-size: 13px !important;
+    font-weight: 700;
+    line-height: 22px;
+    letter-spacing: 0.20000000298023224px;
+    text-align: right;
+    padding: 8px 18px;
+    border-radius: 3px;
+}
+
+.btn--colored {
+    background-color: #1BA0AB;
+    color: white;
+
+}
+
+.btn--big {
+    font-size: 10px !important;
+    display: flex;
+    place-content: center;
+    text-transform: uppercase;
+}
+
+.btn__icon {
+    padding-left: 0.4rem;
+    margin-top: 0.1rem;
+}
+
+/* Experts Page start here*/
+
+.experts_page {
```

Comment:
try to use the same naming convention, above you used kebab case, and here you're using snake_case